doc/nrf/app_dev/device_guides/wifi_coex.rst

```diff
@@ -269,6 +249,7 @@ The *DT property* is the name of the devicetree node property that configures th
 
 In cases where the GPIO is asserted after the radio activity has begun, the ``GRANT`` signal triggers a software interrupt, which in turn disables the radio.
 No guarantee is made on the latency of this interrupt, but the ISR priority is configurable.
+A priority as close to 0 as possible is recommended to avoid delaying the coex interface.
```

Comment:
```suggestion
A GPIOTE interrupt priority as close to 1 as possible is recommended to avoid delaying the coex interface.
```
Perhaps this makes it clearer that it's a priority >0 and it's not a property of the coex pin?